src/track/serato/markers2.cpp

```diff
@@ -639,7 +641,8 @@ QList<CueInfo> SeratoMarkers2::getCues() const {
                 pLoopEntry->getEndPosition(),
                 pLoopEntry->getIndex(),
                 pLoopEntry->getLabel(),
-                std::nullopt); // Serato's Loops don't have a color
+                std::nullopt, // Serato's Loops don't have a color
```

Comment:
What happens in the case if one imports Serato loops, sets a color and imports Serato loops again? 
Please add a comment about this.

Reply:
If you reimport from the metadata, all changes you made inside Mixxx are lost. That doesn't just apply to loop colors, it applies to everything. I wouldn't know where to place such a comment.

Reply:
Ah I understand. 

So the is an "issue" that we loose the colour on ever round trip.
One might think that the color in Mixxx is preserved. 

My idea was to add here a note: 

// Note: the loop color set in Mixxx will be lost after a round trip.   

But I am not sure if here is the right place.

Reply:
> So the is an "issue" that we loose the colour on ever round trip.
> One might think that the color in Mixxx is preserved.

With this implementation: Yes.

Some more information: The loops do actually have something that look like the color field, which is always set to `#27AAE1`. However, it's not possible to change the loop color in Serato and the manually changing the color in the Metadata has no effect (field is still blue).

Right now I just implemented the conservative approach and expect the field to have that exact value, then set the color of the imported cue to nullopt.

Instead, we could also import/export the color for loops. The downside is that

1. The loop color will be blue in Serato anyway, because Serato ignores it
2. We'd write Serato Metadata values that are impossible to create using Serato (because Serato doesn't support changing the loop color).

The advantage would be that if you use Serato Metadata to transfer cues from one Mixxx instance to another, the loop color will not be lost.

It's just a few lines of code, so I could easily add support for importing the loop color. Might be a good idea. Should I?

Reply:
No, I think the conservative approach is a good idea here.